File: TCDNew-master/src/ui.android/lib/models/dtos/ProductDto.dart
class ProductDto {
  String id;
  String name;
  String abc;
  List<String> barcodes;

  ProductDto(this.id, this.name, this.abc, this.barcodes);

  factory ProductDto.fromJson(Map<String, dynamic> json) {
    ProductDto data;

    if (json != null) {
      final barcodes = List<String>();
      json['barcodes'].forEach((value) => barcodes.add(value));

      data = new ProductDto(
        json['id'],
        json['name'],
        json['abc'],
        barcodes
      );
    }

    return data;
  }
}

File: TCDNew-master/src/api/Cen.Wms.Domain.Purchase.Api/Dtos/ProductDto.cs
﻿namespace Cen.Wms.Domain.Purchase.Api.Dtos
{
    public class ProductDto
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Abc { get; set; }
        public string[] Barcodes { get; set; }
    }
}
